failed to merge directives which has parameter as enum

```
directive @access(
  requiredProjectId: Boolean
  subject: String
  ability: PermissionAbility
) on FIELD_DEFINITION

type Mutation {
  createProjectRole(title: String!): ProjectRole!
    @access(
      requiredProjectId: true
      subject: "Project"
      ability: DELETE
    )
}
```


this is error

```
panic: lib.Error("/backend/src/graphql/modules/project-role/schema.graphql:13:5: unexpected ")"")

goroutine 1 [running]:
github.com/mununki/gqlmerge/lib.errorf(...)
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/parse.go:14
github.com/mununki/gqlmerge/lib.(*lexer).consumeIdent(0x140001e5b20, {0x0, 0x0, 0x90?})
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/lex.go:486 +0x27c
github.com/mununki/gqlmerge/lib.(*Parser).parseDirectives(0x140001d3820)
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/parse.go:132 +0x220
github.com/mununki/gqlmerge/lib.(*Schema).Parse(0x140001e81c0, 0x140001d3820)
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/schema.go:452 +0x2f94
github.com/mununki/gqlmerge/lib.parseSchema({0x100ffaadf?, 0x1009d8f80?})
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/merge.go:51 +0xdc
github.com/mununki/gqlmerge/lib.Merge({0x1010e7790, 0x1}, {0x140001d3ac8, 0x1, 0x1?})
	/go/pkg/mod/github.com/mununki/gqlmerge@v0.2.15/lib/merge.go:19 +0x9c
main.mergeGql()
	/backend/main.go:30 +0x48
main.main()
	/backend/main.go:46 +0x248
exit status 2
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## v0.2.16(unreleased)
 
+- Fix parse directive with space https://github.com/mununki/gqlmerge/pull/56
+
 ## v0.2.15
 
 - Fix parse and write the default values in the input value definition https://github.com/mununki/gqlmerge/pull/53

--- a/lib/parse.go
+++ b/lib/parse.go
@@ -153,6 +153,7 @@ func (p *Parser) parseDirectives() []*Directive {
 				if p.lex.peek() == ',' {
 					p.lex.consumeToken(tokComma)
 				}
+				p.lex.skipSpace()
 			}
 			p.lex.consumeToken(tokRParen)
 		}

--- a/test/directives/generated.graphql
+++ b/test/directives/generated.graphql
@@ -1,3 +1,10 @@
+directive @access(
+        requiredProjectId: Boolean
+        subject: String
+        ability: PermissionAbility
+    ) on FIELD_DEFINITION
+
+
 type Person @paint {
     name: String
     age: Int
@@ -7,6 +14,12 @@ type Person @paint {
 type ExampleType implements Node @deprecated {
     id: ID
     oldField: String
+}
+
+type Mutation {
+    createProjectRole(title: String!): ProjectRole! @access(        requiredProjectId: true
+        subject: "Project"
+        ability: DELETE)
 }
 
 

--- a/test/directives/schema/object.graphql
+++ b/test/directives/schema/object.graphql
@@ -8,3 +8,18 @@ type ExampleType implements Node @deprecated {
   id: ID
   oldField: String
 }
+
+directive @access(
+  requiredProjectId: Boolean
+  subject: String
+  ability: PermissionAbility
+) on FIELD_DEFINITION
+
+type Mutation {
+  createProjectRole(title: String!): ProjectRole!
+    @access(
+      requiredProjectId: true
+      subject: "Project"
+      ability: DELETE
+    )
+}

